package.json

```diff
@@ -26,6 +26,7 @@
     "plaiceholder": "^2.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-error-boundary": "^4.0.4",
```

Comment:
don't install this since u ended up just making your own `ErrorBoundary` component